material/partials/languages/kn.html

```diff
@@ -0,0 +1,58 @@
+{#-
+  This file was automatically generated - do not edit
+-#}
+{% macro t(key) %}{{ {
+    "language": "kn",
+    "action.edit": "ಈ ಪುಟವನ್ನು ತಿದ್ದುಪಡಿ ಮಾಡಿ",
+    "action.skip": "ವಿಷಯಕ್ಕೆ ತೆರಳಿ",
+    "action.view": "ಈ ಪುಟದ ಮೂಲವನ್ನು ವೀಕ್ಷಿಸಿ",
+    "announce.dismiss": "ಇದನ್ನು ಮತ್ತೊಮ್ಮೆ ತೋರಿಸಬೇಡಿ",
+    "blog.archive": "ಹಳೆಯ ಲೇಖನ",
+    "blog.categories": "ವರ್ಗಗಳು",
+    "blog.categories.in": "ರಲ್ಲಿ",
+    "blog.continue": "ಓದು ಮುಂದುವರೆಸಿ",
+    "blog.draft": "ಆರಂಭಿಕ ಬರವಣಿಗೆ",
+    "blog.index": "ಸೂಚ್ಯಂಕಕ್ಕೆ ಹಿಂತಿರುಗಿ",
+    "blog.meta": "ಮಾಹಿತಿಯ ಬಗ್ಗೆ ಮಾಹಿತಿ",
+    "blog.references": "ಸಂಬಂಧಿತ ಉಲ್ಲೇಖಗಳು",
+    "clipboard.copy": "ಇದನ್ನು ನಕಲಿಸಿ",
+    "clipboard.copied": "ಇದನ್ನು ನಕಲು ಮಾಡಿದೆ",
+    "consent.accept": "ನಾನು ಇದನ್ನು ಒಪ್ಪಿಕೊಳ್ಳುತ್ತೇನೆ",
+    "consent.manage": "ಸಂರಚನೆಯನ್ನು ನಿರ್ವಹಿಸಿ",
+    "consent.reject": "ನಾನು ಇದನ್ನು ತಿರಸ್ಕರಿಸುತ್ತೇನೆ",
+    "footer": "ಅಡಿಟಿಪ್ಪಣಿ",
+    "footer.next": "ಮುಂದಿನ ಸಂಚಿಕೆ",
+    "footer.previous": "ಹಿಂದಿನ ಸಂಚಿಕೆ",
+    "header": "ಮೇಲ್ಟಿಪ್ಪಣಿ",
+    "meta.comments": "ಪ್ರತಿಕ್ರಿಯೆಗಳು",
+    "meta.source": "ಮೂಲ",
+    "nav": "ಸಂಚರಣೆ",
+    "readtime.one": "ಓದಲು ೧ ನಿಮಿಷ ತೆಗೆದುಕೊಳ್ಳುತ್ತದೆ",
+    "readtime.other": "ಓದಲು # ನಿಮಿಷಗಳನ್ನು ತೆಗೆದುಕೊಳ್ಳುತ್ತದೆ",
+    "rss.created": "ಆರ್ಎಸ್ಎಸ್ ಸೇವೆ",
+    "rss.updated": "ಆರ್ಎಸ್ಎಸ್ ಸೇವೆಯಿಂದ ಇತ್ತೀಚಿನ ನವೀಕರಣ",
+    "search": "ಹುಡುಕಿ",
+    "search.config.lang": "kn",
```

Comment:
Have you actually tested whether Material for MkDocs works with those values? [`lunr-languages`](https://github.com/MihaiValentin/lunr-languages) has _very_ limited support for Indian languages. My guess is that there will be errors in the console, no search, and other effects.

Reply:
Please accept my apologies. I misunderstood that it was just as simple as adding the jinja file, build the project and check if the languages is reflected by changing theme language at `mkdocs.yml`.

Anyway, let me look at this and try to make necessary changes.

Reply:
In my opinion there must be short info of `lunr-languages` in contributions guide's missing translation section.

Reply:
Raised PR to that would add the language support on [lunr-languages](https://github.com/MihaiValentin/lunr-languages/pull/100).